source/vstd/std_specs/hash.rs

```diff
@@ -684,6 +905,19 @@ pub assume_specification<Key, S>[ HashSet::<Key, S>::clear ](m: &mut HashSet<Key
         m@ == Set::<Key>::empty(),
 ;
 
+pub assume_specification<'a, Key, S>[ HashSet::<Key, S>::iter ](m: &'a HashSet<Key, S>) -> (r: Iter<
+    'a,
+    Key,
+>)
+    ensures
+        obeys_key_model::<Key>() && builds_valid_hashers::<S>() ==> {
```

Comment:
Is it intended that `obeys_key_model` and `builds_valid_hashers` is here for `HashSet` but not in the corresponding `HashMap` declaration?